pkgs/development/tools/build-managers/buildbot/plugins.nix

```diff
@@ -26,14 +26,14 @@ in {
   www = pythonPackages.buildPythonPackage rec {
     name = "${pname}-${version}";
     pname = "buildbot_www";
-    version = "0.9.0.post1";
+    version = "0.9.3";
 
     # NOTE: wheel is used due to buildbot circular dependency
     format = "wheel";
 
     src = fetchurl {
-      url = "https://pypi.python.org/packages/02/d0/fc56ee27a09498638a47dcc5637ee5412ab7a67bfb4b3ff47e041f3d7b66/${name}-py2-none-any.whl";
```

Comment:
seems to get a 404 with fetchPypi:



`-    src = fetchurl {`
`-      url = "https://pypi.python.org/packages/02/d0/fc56ee27a09498638a47dcc5637ee5412ab7a67bfb4b3ff47e041f3d7b66/${name}-py2-none-any.whl";`
`-      sha256 = "14ghch67k6090736n89l401swz7r9hnk2zlmdb59niq8lg7dyg9q";`
`+    src = pythonPackages.fetchPypi {`
`+      inherit pname version format;`
`+      sha256 = "1yggg6mcykcnv41srl2sp2zwx2r38vb6a8jgxh1a4825mspm2jf7";
`
this results in:

> curl: (22) The requested URL returned error: 404 
> error: cannot download buildbot_www-0.9.3-py2.py3-none-any.whl from any mirror
> builder for ‘/nix/store/3kscl522lv3n73dlcskjlr4rxckaldkb-buildbot_www-0.9.3-py2.py3-none-any.whl.drv’ failed with exit code 1